front/app/containers/Admin/projects/project/permissions/components/ModeratorListRow.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Text, Box } from '@citizenlab/cl2-component-library';
+import Avatar from 'components/Avatar';
+import Button from 'components/UI/Button';
+import { Row } from 'components/admin/ResourceList';
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+import { isAdmin } from 'services/permissions/roles';
+import { IUserData } from 'services/users';
+import styled from 'styled-components';
+import { deleteProjectModerator } from 'services/projectModerators';
+
+interface Props {
+  isLastItem: boolean;
+  moderator: IUserData;
+  projectId: string;
+}
+
+const PendingInvitation = styled.span`
+  font-style: italic;
+`;
+
+const UnknownName = styled.span`
```

Comment:
We could probably use one const for `PendingInvitation` and `UnknownName` since the styles are similar

Reply:
Yeah, wanted to use Text, but I couldn't easily pass font-style so I'll log a ticket to add that. Just copied existing code.